infrastructure/Makefile

```diff
@@ -57,8 +62,8 @@ sandbox-clean:
 # clean up env (removing stopped docker containers, images)
 	$(SANDBOX_PATH)/sandbox clean
 
-sandbox-algod:
-# enter algod's docker shell
+sandbox-shell:
```

Comment:
`sandbox-algod` was named because there are three bash shells for [sandbox](https://github.com/algorand/sandbox#usage): `algod`, `indexer`, `indexer-db`. Can we rename this to `sandbox-algod-shell`?

Reply:
yes, good idea. thanks.